plugins/binaryninja/README.rst

```diff
@@ -25,7 +25,7 @@ MUI requires a copy of Binary Ninja with a GUI. We are currently developing agai
 
 Manticore only operates on native binaries within a Linux environment. EVM support has only been tested on Mac and Linux, and it requires the installation of `ethersplay <https://github.com/crytic/ethersplay>`_.
 
-Python dependencies are currently managed using ``binjastub/requirements.txt`` and ``requirements-dev.txt``. You can run ``make init`` to set up a development environment.
+Python dependencies are currently managed using ``binjastub/requirements.txt`` and ``requirements-dev.txt``. You can run ``just init`` to set up a development environment.
```

Comment:
Could you also add the `just` https://github.com/casey/just tool to the `Requirements` section above this please.